test/testdata/android_basic_test/input.bzl

```diff
@@ -13,8 +14,10 @@ def exercise_the_api():
 exercise_the_api()
 
 def my_rule_impl(ctx):
+    _ignore = [ctx]  # @unused
```

Comment:
I would've thought a var named `_ignore` would be exempt from needing a separate `@unused` declaration.

Reply:
Not any more, unfortunately.